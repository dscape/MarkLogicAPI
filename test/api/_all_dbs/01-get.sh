#!/bin/sh
curl -s localhost:6984/_all_dbs \
  | ruby -e "puts eval(STDIN.read).member?('Documents')"