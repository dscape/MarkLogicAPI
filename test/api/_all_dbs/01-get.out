["Security", "Schemas", "Documents", "App-Services", "Last-Login", "Triggers", "Modules", "Fab"]
 